AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4183,31 +4159,30 @@ function Debug-AzStorageAccountEntraKerbAuth {
                         # If this is the first failure detected, print "FAILED"
                         if ($success)
                         {
+                            Write-TestingFailed -Message ""

```

Comment:
Suggest writing a couple of words here, just for consistency, since we always write a title in our failures.

```suggestion
                            Write-TestingFailed -Message "Fiddler proxy detected"
```